aspnetcore/blazor/hosting-models.md

```diff
@@ -22,10 +22,11 @@ The principal hosting model for Blazor is running client-side in the browser on
 
 ![Blazor client-side: The Blazor app runs on a UI thread inside the browser.](hosting-models/_static/client-side.png)
 
-To create a Blazor app using the client-side hosting model, use either of the following templates:
+To create a Blazor app using the client-side hosting model, use the following template:
```

Comment:
Let's try to avoid the dreaded *single bullet list*. :smile:

```suggestion
To create a Blazor app using the client-side hosting model, use the **Blazor WebAssembly App** template ([dotnet new blazorwasm](/dotnet/core/tools/dotnet-new)).
```
```

Reply:
Also, we need to get the `--hosted` (is that correct?) switch in here. You might want to keep the list but recast it a bit so that the 1st bullet is for pure client-side and the 2nd is for client-side hosted.